server/models/ride.ts

```diff
@@ -15,12 +15,14 @@ export enum Status {
   ARRIVED = 'arrived',
   PICKED_UP = 'picked_up',
   COMPLETED = 'completed',
+  NO_SHOW = 'no_show',
 }
 
 export type RideType = {
   id: string,
   type: Type,
-  status: Status
+  status: Status,
+  late?: boolean,
```

Comment:
Is it possible to have this not optional and default to false upon ride creation? Could make things easier when accessing the endpoint from clients because we won't have to do null checks or something like that.

Reply:
yeah thats true, it should be possible 👍